features/conversation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Conversation
+    As a user 
+    in order to have a conversation with other people 
+    I should be able to send and receive message from others
+
+    Background: User is Logged in and sends message
+        Given the following user is in database
+        | email              | password   | name |
+        | tore@example.com   | foobarbar  | Tore |
+        | olof@example.com   | mypassword | Olof |
+        Given I should be logged in as "Tore"
```

Comment:
You can refactor the next 4 steps and do all of this with a table
```
Given these mails exists
| sender | receiver | subject | body  |
| Tore     | Olof       | Hello    | World |
```
Then in the step definition you use the table information to send the email